src/main/java/chess/dao/ChessGameDAO.java

```diff
@@ -0,0 +1,158 @@
+package chess.dao;
+
+import chess.domain.game.WebChessGame;
+import chess.domain.piece.Color;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class ChessGameDAO {
+
+    private final BoardSerializer boardSerializer = new BoardSerializer();
+
+    public Connection getConnection() {
+        Connection con = null;
+        String server = "localhost:3306"; // MySQL 서버 주소
+        String database = "java_chess"; // MySQL DATABASE 이름
+        String option = "?useSSL=false&serverTimezone=UTC";
+        String userName = "root"; //  MySQL 서버 아이디
+        String password = "root"; // MySQL 서버 비밀번호
+
+        driverLoad();
+        con = driverConnect(con, server, database, option, userName, password);
+        return con;
+    }
+
+    private Connection driverConnect(Connection con, String server, String database, String option,
+        String userName, String password) {
+        try {
+            con = DriverManager
+                .getConnection("jdbc:mysql://" + server + "/" + database + option, userName,
+                    password);
+            System.out.println("정상적으로 연결되었습니다.");
+        } catch (SQLException e) {
+            System.err.println("연결 오류:" + e.getMessage());
+            e.printStackTrace();
+        }
+        return con;
+    }
+
+    private void driverLoad() {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            System.err.println(" !! JDBC Driver load 오류: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+
+    // 드라이버 연결해제
+    public void closeConnection(Connection con) {
+        try {
+            if (con != null) {
+                con.close();
+            }
+        } catch (SQLException e) {
+            System.err.println("con 오류:" + e.getMessage());
+        }
+    }
+
+    public void addGame(WebChessGame chessGame) {
+        String query = "INSERT INTO chess_game (turn, finished, board) VALUES (?, ?, ?)";
+        try (PreparedStatement pstmt = getConnection().prepareStatement(query)) {
```

Comment:
Connection도 닫아주세요 :) 
```suggestion
        try (Connection connection = getConnection(); PreparedStatement pstmt = connection.prepareStatement(query)) {
```

Reply:
아 모두 분리해서 해야하는군요. 감사합니다😭